monai/networks/nets/flexible_unet.py

```diff
@@ -0,0 +1,272 @@
+            if i < skip_connect:
+                skip = skips[i]
+            else:
+                skip = None
+            x = block(x, skip)
+
+        return x
+
+
+class SegmentationHead(nn.Sequential):
+    """
+    Segmentation head.
+    This class refers to `segmentation_models.pytorch
+    <https://github.com/qubvel/segmentation_models.pytorch>`_.
+
+    Args:
+        dim: number of spatial dimensions.
+        in_channels: number of input channels for the block.
+        out_channels: number of output channels for the block.
+        kernel_size: kernel size for the conv layer.
+        act: activation type and arguments.
+        scale_factor: multiplier for spatial size. Has to match input size if it is a tuple.
+
+    """
+
+    def __init__(
+        self,
+        dim: int,
+        in_channels: int,
+        out_channels: int,
+        kernel_size: int = 3,
+        act: Optional[Union[Tuple, str]] = None,
+        scale_factor: float = 1.0,
+    ):
+
+        conv_layer = Conv[Conv.CONV, dim](
+            in_channels=in_channels, out_channels=out_channels, kernel_size=kernel_size, padding=kernel_size // 2
+        )
+        up_layer: nn.Module = nn.Identity()
+        if scale_factor > 1.0:
+            up_layer = UpSample(
+                spatial_dims=dim,
+                scale_factor=scale_factor,
+                mode="nontrainable",
+                pre_conv=None,
+                interp_mode=InterpolateMode.LINEAR,
+            )
+        if act is not None:
+            act_layer = get_act_layer(act)
+        else:
+            act_layer = nn.Identity()
+        super().__init__(conv_layer, up_layer, act_layer)
+
+
+class FlexibleUNet(nn.Module):
+    """
+    A flexible implement of UNet, in which the backbone can be replaced with any efficient network.
+    """
+
+    def __init__(
+        self,
+        in_channels: int,
+        out_channels: int,
+        backbone: str = "efficientnet-b0",
+        pretrained: bool = True,
+        decoder_channels: Tuple = (256, 128, 64, 32, 16),
+        spatial_dims: int = 2,
+    ) -> None:
+        super().__init__()
+
+        if backbone not in efficientnet_feature_channel:
+            raise ValueError(
+                f"invalid model_name {backbone} found, must be one of {efficientnet_feature_channel.keys()}."
+            )
+
+        if spatial_dims not in (2, 3):
+            raise ValueError("spatial_dims can only be 2 or 3.")
+
+        if "ap" in backbone:
+            adv_prop = True
+        else:
+            adv_prop = False
+
+        self.backbone = backbone
+        self.spatial_dims = spatial_dims
+        model_name = backbone
+        encoder_channels = _get_encoder_channels_by_backbone(backbone, in_channels)
+        self.encoder = EfficientNetBNFeatures(
+            model_name=model_name,
+            pretrained=pretrained,
+            in_channels=in_channels,
+            norm=("batch", {"eps": 1e-3, "momentum": 0.1}),
```

Comment:
Hi @binliunls , here could the parameter of `norm` be an argument, thus users can adjust it. Same for `act` and `norm` of decoder. You can refer to: https://github.com/Project-MONAI/MONAI/blob/dev/monai/networks/nets/autoencoder.py#L107

Reply:
Hi, @yiheng-wang-nv ，
    I have modified the code to make norm, active and other parameters to be an argument so that users can change them according to their demand.